app/services/UserService.py

```diff
@@ -19,15 +25,64 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     if model.mail and await UserInDB.all().filter(mail=model.mail).count() != 0:
         raise HTTPException(status_code=409, detail=CommonErrorMessages.MAIL_ALREADY_USED)
-    
+
     if model.login and await UserInDB.all().filter(login=model.login).count() != 0:
```

Comment:
You can refactor this line with the method you used in create_user.
UserInDB.filter(login=model.login).exists()